x-pack/plugins/ml/public/explorer/explorer_controller.js

```diff
@@ -355,23 +354,44 @@ module.controller('MlExplorerController', function (
       clearSelectedAnomalies();
     } else {
       const timerange = getSelectionTimeRange(cellData);
+      $scope.cellData = cellData;
 
-      if (cellData.fieldName === undefined) {
-        // Click is in one of the cells in the Overall swimlane - reload the 'view by' swimlane
-        // to show the top 'view by' values for the selected time.
-        loadViewBySwimlaneForSelectedTime(timerange.earliestMs, timerange.latestMs);
-        $scope.viewByLoadedForTimeFormatted = moment(timerange.earliestMs).format('MMMM Do YYYY, HH:mm');
-      }
+      if (cellData.score > 0) {
+        if (cellData.fieldName === undefined) {
+          // Click is in one of the cells in the Overall swimlane - reload the 'view by' swimlane
+          // to show the top 'view by' values for the selected time.
+          loadViewBySwimlaneForSelectedTime(timerange.earliestMs, timerange.latestMs);
+          $scope.viewByLoadedForTimeFormatted = moment(timerange.earliestMs).format('MMMM Do YYYY, HH:mm');
+        }
 
-      const jobIds = (cellData.fieldName === VIEW_BY_JOB_LABEL) ?
-        cellData.laneLabels : $scope.getSelectedJobIds();
-      const influencers = getSelectionInfluencers(cellData);
+        const jobIds = (cellData.fieldName === VIEW_BY_JOB_LABEL) ?
+          cellData.laneLabels : $scope.getSelectedJobIds();
+        const influencers = getSelectionInfluencers(cellData);
 
-      $scope.cellData = cellData;
-      loadAnomaliesTableData();
+        loadAnomaliesTableData();
+
+        const args = [jobIds, influencers, timerange.earliestMs, timerange.latestMs];
```

Comment:
why is this `args` array used when everything can be passed straight to `loadDataForCharts` ?